wp-content/plugins/jetpack/_inc/build/custom-css/custom-css/js/core-customizer-css.core-4.9.min.js

```diff
@@ -0,0 +1 @@
+!function(s,e){function n(s){var n=e.control.value(s);return n?n.container:null}s(document).ready((function(){var t=n("jetpack_css_mode_control");t&&t.prepend('<span class="customize-control-title">'+window._jp_css_settings.l10n.mode+"</span>");var o=n("jetpack_mobile_css_control");o&&o.prepend('<span class="customize-control-title">'+window._jp_css_settings.l10n.mobile+"</span>");var i=n("wpcom_custom_css_content_width_control");i&&(i.append('<span class="description">'+window._jp_css_settings.l10n.contentWidth+"<span>"),i.find("input").after("<span>px</span>")),s("<div />",{id:"css-help-links",class:"css-help"}).appendTo(n("custom_css")),s("<a />",{id:"help-link",target:"_blank",rel:"noopener noreferrer",href:window._jp_css_settings.cssHelpUrl,text:window._jp_css_settings.l10n.css_help_title}).prependTo("#css-help-links"),window._jp_css_settings.areThereCssRevisions&&s("<a />",{id:"revisions-link",target:"_blank",rel:"noopener noreferrer",href:window._jp_css_settings.revisionsUrl,text:window._jp_css_settings.l10n.revisions}).prependTo("#css-help-links"),e("jetpack_custom_css[preprocessor]",(function(s){s.bind((function(s){var n={default:"text/css",less:"text/x-less",sass:"text/x-scss"},t="text/css";void 0!==n[s]&&(t=n[s]),e.control("custom_css").deferred.codemirror.done((function(s){s.setOption("mode",t),"text/css"===t?s.setOption("lint",!0):s.setOption("lint",!1)}))}))}))}))}(jQuery,window.wp.customize);
```

Comment:
:warning: **Warning( severity 5 )**: File appears to be minified and cannot be processed (*Internal.Tokenizer.Exception*).